PubnativeLite/PubnativeLite/MRAID/HyBidMRAIDView.m

```diff
@@ -793,6 +793,9 @@ - (void)addContentInfoViewToView:(UIView *)view {
     contentInfoView.delegate = self;
     [view addSubview:contentInfoViewContainer];
     [contentInfoViewContainer addSubview:contentInfoView];
+    
+    [[HyBidViewabilityWebAdSession sharedInstance] addFriendlyObstruction:contentInfoViewContainer toOMIDAdSession:adSession withReason:@"This view is related to Content Info" isInterstitial:isInterstitial];
```

Comment:
@faresbh Just to double check, we are sure that we have a valid and not nil `adSession` there right? Because in the previous place where this code piece sat, we first create an `adSession` and then add all the friendly obstructions.

Reply:
I think it's better to add the check, to be on the safe side.

Reply:
through this change should be also added to development, and the validation branch as well

Reply:
Yeah @faresbh I was just about to write that we should add the check to the other branches as well. 😂 Sorry to asked that question a bit late but it just came to me right now. 😅

Reply:
adding that check, make the viewbility back to 99%

Reply:
i think the friendly obstruction is added before the session is created. and that will not harm unless the viewbility is less than 100.

Reply:
So maybe it is not adding it at all then? 🤔 Are we seeing this obstruction in Charles or etc.? To make sure that it is added to the session properly? I got a feeling that adding the friendly obstruction there, has no effect and behaves as if we have not added it. If there is no `adSession` to add, where it is adding it? 😂

Reply:
If I remember correctly, the OMSDK documentation also suggests to add any friendly obstruction after having a valid `adSession`

Reply:
For sure it's added. otherwise, the viewability tracking will be unchanged.